press.js

```diff
@@ -1,25 +1,26 @@
 export function pressKeyUX(pressedClass) {
-  let pressed
+  let pressedElement
+  const classes = pressedClass.split(' ');
 
-  function keyUp() {
-    if (pressed) {
-      pressed.classList.remove(...pressedClass.split(' '))
+  function handleKeyUp() {
+    if (pressedElement) {
+        pressedElement.classList.remove(classes)
     }
   }
 
-  function click(event) {
+  function handleClick(event) {
     if (event.clientX === 0 && event.clientY === 0) {
-      event.target.classList.add(...pressedClass.split(' '))
-      pressed = event.target
+      event.target.classList.add(classes)
+      pressedElement = event.target
     }
   }
 
   return window => {
-    window.addEventListener('keyup', keyUp)
-    window.addEventListener('click', click)
+      window.addEventListener('keyup', handleKeyUp)
```

Comment:
Can you revert indent changes?

Reply:
Done, replace const with let for consistency